ppx/test/signature.t/run.t

```diff
@@ -49,7 +49,7 @@
   module MyPropIsOptionOptionBool = {
     [@mel.obj]
     external makeProps:
-      (~myProp: option(bool)=?, ~key: string=?, unit) =>
+      (~myProp: bool=?, ~key: string=?, unit) =>
       {. "myProp": option(option(bool))};
```

Comment:
This doesn't look good, myProp shouldn't be `option(option ...`

Reply:
I don't get it? this looks correct in the signature part.